tools/tesseratos/src/tesseratos/main.cpp

```diff
@@ -68,5 +71,20 @@ int main(int argc, char** argv)
             input.bind(*bindings);
         });
 
+    cubos.startupSystem("add some menu items just for testing purposes").call([](Commands cmds) {
```

Comment:
I would maybe change the name of this system besides that lgtm

Reply:
It's not really important, this is code that will be erased as soon as actual menus are added. What do you suggest though?

Reply:
It's not a deal breaker to me, it can stay the same. I just thought it was something that went unnoticed